js/image/src/Webcam.svelte

```diff
@@ -85,6 +88,11 @@
 						name: "sample." + mimeType.substring(6),
 						is_example: false
 					});
+					dispatch("stop_recording", {
+						data: e.target.result,
+						name: "sample." + mimeType.substring(6),
+						is_example: false
+					});
```

Comment:
This is the tweak. If you run the demo you posted, you'll notice is still works as expected.

```suggestion
					dispatch("stop_recording");
```